api/nakadi-event-bus-api.yaml

```diff
@@ -54,15 +54,10 @@ info:
     whose consumption tracking is managed by Nakadi, freeing Consumers from any responsibility in
     tracking of the current position on a Stream.
 
-    **Note** *Currently the high level API is out of scope in this specification.  It is in the
-    short term plan to be included.*
-
 
     Scope and status of the API
     ---------------------------------
 
-    The API specification is in **draft** state and is subject to change.
```

Comment:
I decided to remove this comment since it makes no sense to have software in production for months but state that it's api is still in draft. I think people can rely on our api for been stable and evolve in a compatible manner.